code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -276,16 +276,18 @@
 	. = ..()
 
 /obj/item/weapon/gun/projectile/leyon
-	name = "10mm pistol"
-	desc = "The Leyon LCC Everyman is a small pistol that holds five shots and is loaded with a stripper clip, popular for self-defense on Mars. Uses 10mm rounds."
+	name = "Leyon pistol"

```

Comment:
All guns, at least the great majority of the ballistics follow this format, the brand or etc should stay on the desc.

Reply:
Indeed, but we already have one that is called "10mm pistol" (/obj/item/weapon/gun/projectile/tanto)

Reply:
I guess that the name of the tanto, that can fire in burst, could be changed to indicate the automatic feature it has.

Reply:
Yes. We already have a 10mm pistol. Not really sure why we needed another one.

Reply:
The reasoning for it has been explained in the suggestion topic that has been created when it was originally implemented
https://forums.aurorastation.org/viewtopic.php?f=21&t=11486